README.md

```diff
@@ -24,6 +24,14 @@ To avoid having to set these manually in your local development environment, [di
 
 The environment variables are documented in the [Skate `.envrc.template` file](.envrc.template).
 
+> [!NOTE]
+> Some of these configuration values are shared between Skate team members.
+> While there are still some values which are not shared and need to be configured in
+> `.envrc.private`, to facilitate easier setup and to reduce the amount of work done when cloning
+> for team members, the [`.envrc`](./.envrc) file is configured to source the shared
+> configuration values from 1Password using the 
+> [.env.1p.skate file](./.env.1p.skate) and the 1Password CLI (which you must also have on your system).
+
```

Comment:
[context: github comment](https://github.com/mbta/skate/pull/2522#discussion_r1544546586) @hannahpurcell: Here's the docs you "asked" for!! I'd appreciate any feedback you have, especially on the "copy". I'm not sold on using a block quote and the `[!NOTE]` callout syntax, but I think it helps when reading it.

Reply:
The `1Password CLI` that we need installed is `op`, right? Should we include a link here to installation instructions? 

This link, I think?
https://developer.1password.com/docs/cli/get-started/

Reply:
@joshlarson updated!